corpora/csip/metadata/dmdsec/CSIP28/testCase.xml

```diff
@@ -3,13 +3,16 @@ XML lists -->
 <testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd"
   testable="TRUE">
   <!-- Unique ID for the test case -->
-  <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP28"/>
+  <id specification="E-ARK CSIP" version="2.1.0" requirementId="CSIP28"/>
   <!-- URL references to requirements for convenient lookup -->
   <references>
     <reference requirementId="CSIP28" URL="http://earkcsip.dilcis.eu/#CSIP28"/>
   </references>
   <!-- The full text of the requirement. -->
-  <requirementText>The date the linked file was created.
+  <requirementText>
+File size
```

Comment:
Seems like a copy paste error from the preceding rule crept in? I believe this should be:

```xml
<description>The creation date and time of the referenced file.</description>
<cardinality>1..1</cardinality>
<level>MUST</level>
```